sys/Commands/Console.php

```diff
@@ -39,6 +41,8 @@ class Console {
 	public function __construct() {
 		self::$instance = $this;
 		$this->stdin = fopen('php://stdin', 'r');
+		$this->timezone = new \DateTimeZone('Europe/Berlin');   // do not use Config::getTimezone(), because Console may run before Config exists
```

Comment:
I think this should be defined in a config file, and not to hard-code the timezone

Reply:
Do you mean in a new config file? Did you see the comment explaining why Config::getTimezone() was not used?

Reply:
This is a hen-and-egg problem...

Reply:
I've read the comment of course. I know that it can run before config file exists, but in the most cases it exists. So it's OK if the hardcoded value is only a fallback (but if fallback, it can be UTC as well). But at least we should set it with a setter if config already exists, and we have a timezone. Timezone is configurable to use the configured value.

Reply:
It's OK to set it from config file and fallback to UTC, thanks.